tidy3d/plugins/mode/solver.py

```diff
@@ -168,6 +170,12 @@ def compute_modes(
             target = mode_spec.target_neff
         target_neff_p = target / np.linalg.norm(kp_to_k) + fp_eps
```

Comment:
let's remove this `fp_eps` shift here then; it maybe was meant to play the same purpose but doesn't always?

Reply:
it might be too small and overflowed.

Reply:
right, so let's remove it in favor of the new code.